Tests/Parser/Client/fixtures/library.yml

```diff
@@ -528,3 +527,33 @@
     type: library
     name: Axios
     version: "1.2.0"
+-
+  user_agent: HTMLParser/1.6
+  client:
+    type: library
+    name: HTML Parser
+    version: "1.6"
+-
+  user_agent: python-urllib3/1.26.9
+  client:
+    type: library
+    name: Python urllib
+    version: 1.26.9
+-
+  user_agent: msray-plus
+  client:
+    type: library
+    name: msray
+    version: ""
+-
+  user_agent: Slim Framework
+  client:
+    type: library
+    name: Slim
```

Comment:
Would not "Slim Framework" be a more appropriate name, since this is their official name?